extension/controller/backgroundScript.js

```diff
@@ -213,6 +214,14 @@ const messageListener = function(message, sender) {
           /*
            * request the experiments API do display the infobar
            */
+
+          let from = "en".concat(message.languageDetection.pageLanguage);
+          let to = message.languageDetection.navigatorLanguage.substring(0,2).concat("en");
+          if (from === "enen") from = to;
```

Comment:
a developer's comment explaining 220 and 221 would be helpful. It's not obvious why "enen" is a special case